packages/core/src/enqueue_links/enqueue_links.ts

```diff
@@ -332,7 +354,7 @@ export async function enqueueLinks(options: SetRequired<EnqueueLinksOptions, 're
         }
 
         // Generate requests based on the user patterns first
-        const generatedRequestsFromUserFilters = createRequests(requestOptions, urlPatternObjects);
+        const generatedRequestsFromUserFilters = createRequests(requestOptions, urlPatternObjects, urlBlacklistPatternObjects);
         // ...then filter them by the enqueue links strategy (making this an AND check)
         return filterRequestsByPatterns(generatedRequestsFromUserFilters, enqueueStrategyPatterns);
```

Comment:
Do you think it's best to do the blacklist filter in between `generatedRequestsFromUserFilters` and the return? We may have better performance if we do it in `createRequests` but code readability goes down a little.

Reply:
lets optimize when we see the need for it